beater/route_config.go

```diff
@@ -228,7 +228,7 @@ func (v v2Route) Handler(url string, c *Config, report publish.Reporter) http.Ha
 	}
 
 	if url == V2RumURL {
-		if rlc, err := NewRlCache(c.RumConfig.EventRate.LruSize, c.RumConfig.EventRate.Limit); err == nil {
+		if rlc, err := NewRlCache(c.RumConfig.EventRate.LruSize, c.RumConfig.EventRate.Limit, burstMultiplier); err == nil {
```

Comment:
I think you should now remove https://github.com/elastic/apm-server/pull/1459/files#diff-48b2aa9b71bf27a37b9122608aa3dfb1R28, and instead create a `v2BurstMultiplier` const here.